launcher/ui/pages/modplatform/ModModel.cpp

```diff
@@ -220,7 +220,8 @@ void ListModel::versionRequestSucceeded(QJsonDocument doc, QString addonId)
     auto& current = m_parent->getCurrent();
     if (addonId != current.addonId) { return; }
 
-    QJsonArray arr = doc.array();
+    auto arr = Json::ensureArray(doc.object(), "data");
```

Comment:
probably this broke it, since modrinth has no data array

Reply:
ahaha okay I see